src/components/Card.tsx

```diff
@@ -26,8 +32,9 @@ const Card: React.FC<IProps> = ({
     borderColor="lightBorder"
     p={6}
     className={className}
+    style={{ justifyContent: "space-between" }}
   >
-    <div>
+    <Container>
```

Comment:
Lets use Chakra components to do this. No need for the `!important` here
```suggestion
    <Box minHeight="67px">
```